zoo/base/management/commands/fake.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+import django
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from ... import settings
+from ....services.models import Service, Status
+
+
+class Command(BaseCommand):
+    help = "Fake data for local dev + creates superuser."
+
+    def handle(self, *args, **options):
+        # todo - is this worth ?
+        if settings.DATABASES["default"]["PASSWORD"] != "postgres":
+            print("You're not on localhost, nothings happening.")
+            sys.exit(1)
+
+        # todo - use faker, create more data ?
+        s = Service(
+            owner="jake",
```

Comment:
I think that if we use `faker` here it could be more useful, so it generates random data, so running the same command multiple times would add more fake services. But not sure about that

Reply:
Oops, just saw your comment 😄

Reply:
:D I can create 100 services (or it can be option how many). Question is - do you see value in doing so ? Should I add faker to requirements and create more services with proper data? 

There's also some Django integration - https://github.com/joke2k/django-faker - but as I read the "guessers" it is not very smart. I think I would rather manually fake the attributes one by one. But maybe its worth to try 🤷‍♂

Reply:
The main limitation I'm seeing is that there aren't repos to relate the services to... so it's not that useful.

To make it more useful I'd probably run `sync_repos` first, and pick up a random repo from `Repositories.objects.all()`, and in that case I can see more value on using the faker, but as you mentioned, just the `owner` and the `name` should be enough for now.

Of course that would require setting up a `GitHub` or `GitLab` token prior to running this command

Reply:
oh I get it. I'll try to come up with something :)